main.nf

```diff
@@ -56,30 +78,66 @@ if (params.help){
 
 // Configurable variables
 params.name = false
-params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
-params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
 params.email = false
 params.plaintext_email = false
 
-multiqc_config = file(params.multiqc_config)
+params.martmap = false
+params.isobaric = false
+params.activation = 'hcd' // Only for isobaric quantification
+params.outdir = 'results'
+params.normalize = false
+params.genes = false
+params.symbols = false
+params.fastadelim = false
+params.genefield = false
+params.quantlookup = false
+params.fractions = false
+params.hirief = false
+params.pipep = false
+params.onlypeptides = false
+params.noquant = false
+
+// Validate and set file inputs
+fractionation = params.hirief || params.fractions ? true : false
+mods = file(params.mods)
+if( !mods.exists() ) exit 1, "Modification file not found: ${params.mods}"
+tdb = file(params.tdb)
+if( !tdb.exists() ) exit 1, "Target fasta DB file not found: ${params.tdb}"
+
+// file templates
+hkconf = file("$baseDir/assets/hardklor.conf")
+
+if (params.martmap) {
+  martmap = file(params.martmap)
+  if( !martmap.exists() ) exit 1, "Biomart ENSEMBL mapping file not found: ${params.martmap}"
+}
+if (params.pipep) {
+  trainingpep = file(params.pipep)
+  if( !trainingpep.exists() ) exit 1, "Peptide pI data file not found: ${params.pipep}"
+}
 output_docs = file("$baseDir/docs/output.md")
 
-// Validate inputs
-if ( params.fasta ){
-    fasta = file(params.fasta)
-    if( !fasta.exists() ) exit 1, "Fasta file not found: ${params.fasta}"
+// Parse input variables values
```

Comment:
This needs more explicit details: What is happening here? Is this a set of defaults but can be set by users?